eduandebruijne/admin-bundle/9.1/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+    "bundles": {
+        "EDB\\AdminBundle\\EDBAdminBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
+    },
+    "composer-scripts": {
+        "assets:install %PUBLIC_DIR%": "symfony-cmd",
+        "cache:clear": "symfony-cmd"
+    },
+    "env": {
+        "ADMIN_PATH": "admin",
+        "MEDIA_PATH": "/",
+        "GOOGLE_CLIENT_ID": "",
+        "GOOGLE_CLIENT_SECRET": ""
```

Comment:
I would prefix all these env vars to avoid any clash in case projects also have the same names (ADMIN_PATH looks very generic for instance).

Reply:
Added `EDB_` as prefix